src/xmipp/applications/scripts/tomogram_reconstruction/tomogram_reconstruction.py

```diff
@@ -231,19 +252,52 @@ def run(self):
         
         self.readInputParams()
         
-        mdTs, ts = self.readTiltSeries(self.fnAngles)
-        
-        tiltAngles = []
-        for objId in mdTs:
-            tiltAngles.append(mdTs.getValue(xmippLib.MDL_ANGLE_TILT, objId))
-        
+        ts = self.readTiltSeries()
+        print(os.path.splitext(self.fnAngles)[1])
+        if os.path.splitext(self.fnAngles)[1] == '.tlt':
+            tiltAngles = self.readTltFile(self.fnAngles)
+        else:
```

Comment:
Maybe it's better to add an elif with .xmd extension comprobation. What if I input a .mp4 shrek 2 movie as the tilt angles file?

Reply:
Lo dejamos a cargo de Xmipp decidir si abre o revienta